Frends.AzureBlobStorage.DeleteContainer/Frends.AzureBlobStorage.DeleteContainer/DeleteContainer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.AzureBlobStorage.DeleteContainer.Definitions;
+using Azure.Storage.Blobs;
+
+namespace Frends.AzureBlobStorage.DeleteContainer
+{
+    public static class AzureBlobStorage
+    {
+        /// <summary>
+        /// Deletes a container from Azure blob storage.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.DeleteContainer)
+        /// </summary>
+        /// <param name="input">Information about the container destination.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Object { string Success }</returns>
+        public static async Task<Result> DeleteContainer(Input input, CancellationToken cancellationToken)
+        {
+            if (input.ConnectionString == null || input.ContainerName == null)
+                throw new ArgumentNullException("Given parameter can't be empty.");
+
+            // get container
+            var container = GetBlobContainer(input.ConnectionString, input.ContainerName);
+
+            if (!await container.ExistsAsync(cancellationToken)) return new Result(true);
```

Comment:
This should return `Result(false)`, since the task operation was not successful.